### ISSUE TITLE
Navigation labels (API integration)

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#581]
Issue group: API
Visibility: 2
Evaluation: Partially Supports
Success Criteria: 2.4.6: Headings and Labels
Needs design: Content

### Description
On the API integrations page, the navigation within the `<main>` element is unlabelled. An appropriate label should be provided to indicate purpose and differentiate from other `<nav>` elements on the page.

![](https://dl.airtable.com/.attachments/8df9d0515d88ba7e77fe49f33792c78a/dfa24959/Capturedecranle2022-03-15a13_40_03.png)

## Related Airtable links
[Link 1](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwBYyaQxOttZ0IAw/recAYwlRcb3Q7daMG?blocks=hide)
[Link 2](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwEUFgBd0KLqH7oo/recdyxF60Q3WUQ2hi?blocks=hide)

**Comment by amazingphilippe** [comment]
Related finding from testing:

https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwBYyaQxOttZ0IAw/recAYwlRcb3Q7daMG?blocks=hide

> GC Notify was difficult to navigate for participants. For a user using a screenreader, the labels weren't descriptive enough (e.g. navigation regions, homepage link). For a screen magnification user, the grey banner on the top nav was too light (but other screen magnification users didn't comment on it)

**Comment by amazingphilippe** [comment]
Provide content for this in both languages

**Comment by amazingphilippe** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/581?planning-poker) with ZenHub @andrewleith

**Comment by amazingphilippe** [comment]
[Figma for labels](https://www.figma.com/file/r3GzRBjBDWvL2AwCWq1e1W/Navigating-the-site-%7C-Naviguer-le-site?node-id=7800%3A4588)

**Comment by andrewleith** [comment]
Looks good!
